manifests/database/postgresql.pp

```diff
@@ -37,9 +37,33 @@
       command     => "liquibase --driver=org.postgresql.Driver \
                             --classpath=/usr/share/java/postgresql-jdbc.jar:/var/lib/${candlepin::tomcat}/webapps/candlepin/WEB-INF/classes/ \
                             --changeLogFile=db/changelog/changelog-create.xml \
-                            --url=jdbc:postgresql:${candlepin::db_name} \
-                            --username=${candlepin::db_user}  \
-                            --password=${candlepin::db_password} \
+                            --url=jdbc:postgresql:${db_name} \
+                            --username=${db_user}  \
+                            --password=${db_password} \
+                            migrate \
+                            -Dcommunity=False \
+                            >> ${candlepin::log_dir}/cpdb.log \
+                            2>&1 && touch /var/lib/candlepin/cpdb_done",
+      creates     => "${candlepin::log_dir}/cpdb_done",
+      refreshonly => true,
+      before      => Service[$candlepin::tomcat],
+      require     => [
+        Package['candlepin'],
+        Concat['/etc/candlepin/candlepin.conf']
+      ],
+    }
+
+    Postgresql::Server::Role[$db_user] -> Postgresql::Server::Database[$db_name]
+  }else{
+
+    exec { 'cpdb':
```

Comment:
Seems like this should be run no matter if manage_db is true or not so should we make one command and move it after the if statement?